alphastats/llm/uniprot_utils.py

```diff
@@ -1,67 +1,69 @@
+import re
 from typing import Dict, List, Union
 
 import requests
 import streamlit as st
 
 from alphastats.gui.utils.ui_helper import StateKeys
 
-uniprot_fields = [
-    # Names & Taxonomy
-    "gene_names",
-    "organism_name",
-    "protein_name",
-    # Function
-    "cc_function",
-    "cc_catalytic_activity",
-    "cc_activity_regulation",
-    "cc_pathway",
-    "kinetics",
-    "ph_dependence",
-    "temp_dependence",
-    # Interaction
-    "cc_interaction",
-    "cc_subunit",
-    # Expression
-    "cc_tissue_specificity",
-    "cc_developmental_stage",
-    "cc_induction",
-    # Gene Ontology (GO)
-    "go",
-    "go_p",
-    "go_c",
-    "go_f",
-    # Pathology & Biotech
-    "cc_disease",
-    "cc_disruption_phenotype",
-    "cc_pharmaceutical",
-    "ft_mutagen",
-    "ft_act_site",
-    # Structure
-    "cc_subcellular_location",
-    "organelle",
-    "absorption",
-    # Publications
-    "lit_pubmed_id",
-    # Family & Domains
-    "protein_families",
-    "cc_domain",
-    "ft_domain",
-    # Protein-Protein Interaction Databases
-    "xref_biogrid",
-    "xref_intact",
-    "xref_mint",
-    "xref_string",
-    # Chemistry Databases
-    "xref_drugbank",
-    "xref_chembl",
-    "reviewed",
-]
+# Fields are only relevant in the context of table output, with json you automatically get everything, with different keys
+# uniprot_fields = [
```

Comment:
could this be removed?

Reply:
If the changes are approved otherwise I would remove it before merging, yes.